cpp/ql/src/experimental/Security/CWE/CWE-089/SqlPqxxTainted.ql

```diff
@@ -0,0 +1,139 @@
+/**
+ * @name Uncontrolled data in SQL query to Postgres
+ * @description Including user-supplied data in a SQL query to Postgres
+ *              without neutralizing special elements can make code
+ *              vulnerable to SQL Injection.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id cpp/sql-injection-via-pqxx
+ * @tags security
+ *       external/cwe/cwe-089
+ */
+
+import cpp
+import semmle.code.cpp.security.Security
+import semmle.code.cpp.dataflow.TaintTracking
+import DataFlow::PathGraph
+
+predicate pqxxTransationClassNames(string class_name, string namespace) {
```

Comment:
We use [`camelCalse`](https://github.com/github/codeql/blob/main/docs/ql-style-guide.md#naming) for variables in QL.